.github/workflows/test.yml

```diff
@@ -443,9 +443,9 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        template:
-        - default.yaml
-        - fedora.yaml
+        include:
+        - {template: default.yaml, ssh-port-forwarder: true}
+        - {template: fedora.yaml, ssh-port-forwarder: false}
```

Comment:
Can we have a comment to clarify that ssh-port-forwarder: false is not required by fedora, but added for the coverage of the tests?

Reply:
sure